build-github-branch

```diff
@@ -5,11 +5,12 @@ class BranchOnRepo {
     String repo
     String branch
     String version
+    String originalBranch = ""
 }
 
 def branchesToBuild = [
     new BranchOnRepo('apache/james-project', 'master', '3.2.0-SNAPSHOT'),
-    new BranchOnRepo('apache/james-project', '3.1.0', '3.1.0')
+    new BranchOnRepo('apache/james-project', '3.1.x', '3.1.0', '3.1.0')
```

Comment:
Isn't it `3.1.0` as base branch too?

Reply:
I was thinking on that, and I think we only want to build _living_ branches.
`3.1.0` will never change, development should be merged on `3.1.x`